src/components/SlideshowHero/_slideshow-hero.scss

```diff
@@ -194,10 +194,22 @@
 }
 
 .slideshow__image {
-  height: 100%;
-  object-fit: cover;
   position: absolute;
   width: 100%;
+
+  @supports (object-fit: cover) {
+    height: 100%;
+    object-fit: cover;
+  }
+
+  @media all and (-ms-high-contrast:none) {
+    top: 50%;
+    transform: translateY(-50%);
+  }
+
+  &.u-loading-preload {
+    filter: blur(20px) brightness(110%);
+  }
```

Comment:
not sure about the nesting... add the `u-loading-preload` selector to https://github.com/wellcometrust/corporate-components/tree/develop/src/assets/styles/50-utilities?

I think if the styles are specific to the `slideshow__image`, then use `&.is-loading-preload`, but if they're more agnostic then add to util partials

Reply:
Done

Reply:
ace